Week2/homework/maartjes-work.js

```diff
@@ -46,8 +46,10 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  return tasks
+    .map(durations => durations.duration)
```

Comment:
`Multiply each duration by a per-hour rate for billing (use €20/hour) and sum it all up.` => first convert the duration to hour rate!
It's not wrong, but doing so makes it more clear!

Reply:
I got that, thanks!